setup.py

```diff
@@ -14,12 +14,12 @@
 setuptools.setup(
     name="notion",
     version="0.0.24",
-    author="Jamie Alexandre",
-    author_email="jamalex+python@gmail.com",
+    author="Vojtech Zavrel",
+    author_email="info@vojtechzavrel.cz",
     description="Unofficial Python API client for Notion.so",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    url="https://github.com/jamalex/notion-py",
+    url="https://github.com/zavrel/notion-py",
     install_requires=install_requires,
     include_package_data=True,
     packages=setuptools.find_packages(),
```

Comment:
Could you revert the edits in this file? These are what get used to push to the main PyPI project, which should refer back here. Thanks!

Reply:
Sorry for that I thougth I made this pull request from feature branch, not from master.